ze/btx_zematching_model.yaml

```diff
@@ -152,6 +152,13 @@
       :type: structure
       :members:
       - :name: ^size$
+  - :set_id: eventMemory_without_hSignalEvent_exit
+    :domain: n = eventMemory_without_hSignalEvent_entry.map { |b| b.name.sub("_entry","_exit") }; exits.filter { |a| n.include?(a.name) }.to_set
```

Comment:
So this is creating a list of all the without hSignalEvent entries with "_exit" replaced, then searching through all exits for ones that actually exist? I guess not all _entry have corresponding exit? Took me a minute to map my head around. I think what is confusing about it, is the n being stored as a temporary, and what is actually "returned" is the second part.

Reply:
> So this is creating a list of all the without hSignalEvent entries with "_exit" replaced, then searching through all exits for ones that actually exist?

Yes, just try to find the associated _exit to the `hSignalEvent_entry`. 

>  I guess not all _entry have corresponding exit? 

In generate case no, for our use-case in THAPI always.

> I think what is confusing about it, is the n being stored as a temporary, and what is actually "returned" is the second part.
- Typical ruby, where the last expression is what is returned. But yeah, if this is a useful partner we can create a metabaebl function call `get_opposite_domain` who far all `_entry`, `_exit` find their associated `_exit, _exit`